_posts/platform/deployment/2000-01-01-deploy-scalingo-from-codeship.md

```diff
@@ -19,10 +22,12 @@ On Codeship:
 
 ```bash
 git fetch --unshallow || true
-git push --force git@scalingo.com:my-app.git ${CI_COMMIT_ID}:master
+git push --force ssh://git@ssh.osc-fr1.scalingo.com:2200/my-app.git ${CI_COMMIT_ID}:master
```

Comment:
You should remind the user about different regions.

Reply:
I updated and pushed my modifications, but they don't show up on GitHub. Can you have a look on your side to see if it's an issue with GitHub you also have?